docs/release-notes/january-2024.md

```diff
@@ -0,0 +1,138 @@
+---
+slug: january-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: January 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for January 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in January 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.1](https://github.com/semgrep/semgrep/releases/tag/v1.55.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.2](https://github.com/semgrep/semgrep/releases/tag/v1.55.2)
+  * [<i class="fas fa-external-link fa-xs"></i>1.56.0](https://github.com/semgrep/semgrep/releases/tag/v1.56.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.57.0](https://github.com/semgrep/semgrep/releases/tag/v1.57.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.58.0](https://github.com/semgrep/semgrep/releases/tag/v1.58.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.0](https://github.com/semgrep/semgrep/releases/tag/v1.59.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep's Visual Studio Code extension now runs natively on Windows machines.
+* Added ability for organizations to test connections to GitHub and GitLab by going to
+  **Settings** > **Source Code Managers**.
+* Projects are removed from SCP when the corresponding GitHub repository is
+  archived on GitHub.
+* **CLI tool**: 
+  * Added color-coded severity icons, such as `❯❯❱`, to the CLI
+  output for findings of known severity.
+  * Metrics sent from the CLI to Semgrep now include a breakdown of the number
+  of findings per product. This helps the Semgrep team understand users better.
```

Comment:
I'm not sure what this means. But I think this was a growth project so maybe @bmahe does?

Reply:
This was [changed in release 1.56.0](https://github.com/semgrep/semgrep/releases/tag/v1.56.0).

Do you mean that the description is too vague?

Reply:
Yeah but it looks like it matches what we said publicly so maybe it's okay. I just didn't know what "This helps the Semgrep team understand users better." really meant :p